file/random_access_file_reader.h

```diff
@@ -102,7 +110,8 @@ class RandomAccessFileReader {
       RateLimiter* rate_limiter = nullptr,
       const std::vector<std::shared_ptr<EventListener>>& listeners = {},
       Temperature file_temperature = Temperature::kUnknown,
-      bool is_last_level = false)
+      bool is_last_level = false, uint32_t async_hist_type = 0,
```

Comment:
I don't know if its necessary to pass these. Are there scenarios where we'd want to use different histogram types? If not, we can simplify by hardcoding it. We do that for ticker stats anyway.